docker/user-testnet.conf

```diff
@@ -1,5 +1,5 @@
 alephium.network.network-type = "testnet"
-alephium.discovery.bootstrap = ["18.221.19.228:9973", "3.142.153.145:9973"]
+alephium.discovery.bootstrap = ["3.68.2.201:9973", "34.236.121.90:9973", "54.252.31.241:9973"]
```

Comment:
can't wait the mainnet so we don't need to change that anymore :)

Reply:
We'll introduce DNS soon to mitigate this pain point.